github-merge.py

```diff
@@ -198,6 +198,9 @@ def print_merge_details(pull, title, branch, base_branch, head_branch, acks):
                 print('* {} {}({}){}'.format(message, ATTR_NAME, name, ATTR_RESET))
         else:
             print('{}Top commit has no ACKs!{}'.format(ATTR_WARN, ATTR_RESET))
+    if message is not None:
+        if '@' in message:
+            print('{}Merge message contains an @!{}'.format(ATTR_WARN, ATTR_RESET))
```

Comment:
nit, could `curl https://github.com/$user` for each possible user handle and in that case fail 🙄

Reply:
That seems like complete overkill, so I'd rather not.